src/main/java/com/google/sps/servlets/FindLocationServlet.java

```diff
@@ -5,6 +5,14 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.FilterOperator;
 
 @WebServlet("/find-location")
 public class FindLocationServlet extends HttpServlet {
```

Comment:
comment all these classes.

something like.

/**
 * This servlet is called by the ad whenever the user clicked on the find location button on the ad.
 */

Reply:
Done!